share/setup.nsi.in

```diff
@@ -1,4 +1,4 @@
-Name "@CLIENT_NAME@ (64-bit)"
+Name "@CLIENT_NAME@"
```

Comment:
Does the message below (written 11 years ago) still make sense now that there's no "64 bit version Windows installer", just a "Windows installer"?

>       MessageBox MB_OK|MB_ICONSTOP "Cannot install 64-bit version on a 32-bit system."

Reply:
Considering the condition that triggers this message, the latter is correct. However, the condition itself may now be outdated.  

This code does not affect the UX, so there's no harm in leaving it as is for now.